### ISSUE TITLE
[Feature request] Changelog when updating

### ISSUE THREAD

**Issue by welpo** [corpnewt/Lilu-and-Friends#6]
I would like to have the option to see a changelog after/before I choose to update the script.

It could be something like:

> Would you like to see the changelog?: (y/n)

It doesn't need to be super detailed (for me), but I would like to know what has changed (and save me the trip to github).

Thanks!

**Comment by corpnewt** [comment]
Hey welpo,

I can look into setting up something like this - the problem I have is that I'm not always terribly verbose when committing changes in general - so having to write something into the latest version of the plugins.json file (where the script compares versions) may fall to the wayside.  For the time being, I'll put this on the back burner unless I find an easy way to do it.

Thanks for the suggestion though,

CorpNewt

**Comment by corpnewt** [comment]
Alright - I added a simplified changelog that pops up when I update the plugins.json.  An example of what it looks like:

```
  #######################################################
 #                Checking for Updates                 #
#######################################################
 
v0.0.29 is available (v0.0.28 installed)
What's New: Added WhateverGreen.kext and simplified changelog

Update? (y/n):
```

Again, this relies on me adding little blurbs about what's changed - so I'm sure some update messages will be better than others :)  Hopefully that will suffice though.
